skimage/transform/_orientation.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+
+
+def _normalize(x):
+    """Normalizes an array.
+
+    Parameters
+    ----------
+    x : array_like
+        Array to normalize.
+
+    Returns
+    -------
+    u : array
+        Unitary array.
+
+    Examples
+    --------
+    >>> x = np.arange(5)
+    >>> uX = _normalize(x)
+    >>> np.isclose(np.linalg.norm(uX), 1)
+    True
+    """
+    u = np.asarray(x)
+
+    norm = np.linalg.norm(u)
+
+    return u / norm
+
+
+def _axis_0_rotation_matrix(unit_vector, indices=None):
+    """Generate a matrix that rotates a vector to coincide with the 0th (y-)
```

Comment:
```suggestion
    """Generate a matrix that rotates a vector to be collinear with axis 0.
```

Reply:
ah, thanks for informing of the correct mathy term! I was unsure of what to call this lol